### ISSUE TITLE
Backdoor on iOS?

### ISSUE THREAD

**Issue by Virgula0** [neoneggplant/EggShell#54]
is it only works on iOS jailbroken if I have understand correctly... Bu if I try to code a backdoor for iOS with Xcode with system(""), there is a warning that warn me that system is deprecated from iOS 8...
It works the same also if there is this warn?

**Comment by neoneggplant** [comment]
lookup nstask for getting rid of the deprecation warning -> https://stackoverflow.com/questions/17976289/executing-shell-commands-with-nstask-objective-c-cocoa
As for the no-ip compatibility, what do you mean exactly? You can port forward your router that your server is running on and have an ip from outside the network connect to your router ip on a port, then forward that connection to an internal ip on your network

**Comment by Virgula0** [comment]
Yah it's right, but you could have a router that doesn't supports static ip and with no-ip you can resolve this problem. Try it...it works also in linux. To implement it in this software in my opnion you have only to remove the checking about integer value when you insert your ip....
About your link, that code with NStask works only on Mac OS or also for cocoa apps on iOS?

And above all your work is very good! :)

**Comment by neoneggplant** [comment]
eggshell does have dns support in replace of ip addresses, also, getfacebook only works with safari, it looks like your using chrome

**Comment by Virgula0** [comment]
I have use getfacebook for get cookies of a session of my Facebook on safari, then I'm trying to hijack my account by using edit cookie extension on chrome..... But I don't understand where insert c_user and xs cookies because there aren't when I visit Facebook.com like the picture upside.

**Comment by neoneggplant** [comment]
oh, you create a new cookie field, click the "+" button and insert the new key values

**Comment by Virgula0** [comment]
Yeah I have already set as you have said but it seems doesn't work .... I don't understand ...

<img width="1440" alt="schermata 2017-07-27 alle 19 07 01" src="https://user-images.githubusercontent.com/16546435/28682975-2c65239a-72ff-11e7-8cad-f26d7f862ea6.png">

**Comment by Virgula0** [comment]
> eggshell does have dns support in replace of ip addresses

It not seems if I try to insert a dns using no-ip eggshell crashes...

<img width="1147" alt="schermata 2017-07-28 alle 16 30 15" src="https://user-images.githubusercontent.com/16546435/28721939-310ebe0c-73b2-11e7-9f6f-0f3bff979e4d.png">

**Comment by Virgula0** [comment]
And what about also to add a key logger command?

**Comment by nerdtron123** [comment]
There’s already a keylogger command

**Comment by 0xfffffff0075e2040** [comment]
No there isn't
